cmake/jana_plugin.cmake

```diff
@@ -223,37 +219,32 @@ macro(plugin_add_event_model _name)
 
     if(NOT EDM4EIC_FOUND)
         find_package(EDM4EIC REQUIRED)
-        set(EDM4EIC_INCLUDE_DIR ${EDM4EIC_DIR}/../../include)
     endif()
 
-    # Add include directories
-    # (same as target_include_directories but for both plugin and library)
-    # ${podio_BINARY_DIR} is an include path to datamodel_glue.h
-    plugin_include_directories(${PLUGIN_NAME} SYSTEM PUBLIC ${podio_INCLUDE_DIR} ${EDM4EIC_INCLUDE_DIR} ${EDM4HEP_INCLUDE_DIR} ${podio_BINARY_DIR})
-
```

Comment:
The `${podio_BINARY_DIR}` was intended for `#include <datamodel_glue.h>` to work.

Reply:
Yeah, I see that now, but it's confusing. We should not redefine the `podio` project in `src/services/io/podio` when we also include a target `podio` that sets its own variables `podio_*`. I'd rather we called this internal project something else that is more explicit, like `podio_datamodel_glue` or so. But then it differs from the plugin name, so let's just keep this as is.

Reply:
Maybe `project(datamodel)` is sufficient to make this separation clearer. The difference in name with the plugin does not cause problems.

Reply:
https://github.com/eic/EICrecon/pull/720/commits/cd869e67d502f340a37f58c09e7f3e34c439abba